codespell_lib/_codespell.py

```diff
@@ -984,6 +985,12 @@ def parse_file(
     return bad_count
 
 
+def flatten_comma_separated_arguments(
+    arguments: List(str),
+) -> List(str):
```

Comment:
These changes are great!  Simple but effective -- well done.

Add a doctest?
```suggestion
) -> List(str):
    """
    >>> flatten_comma_separated_arguments(None)
    []
    """
```

Reply:
Yes, I'll look into it. This is just a draft, I'm not even sure it works well when an option is missing. I need tests.

Reply:
Thank you for helping out, by the way. I hope this change will actually decrease code complexity.